wgpu-hal/src/gles/adapter.rs

```diff
@@ -543,13 +544,18 @@ impl super::Adapter {
             #[cfg(any(native, Emscripten))]
             {
                 features.insert(wgt::Features::TEXTURE_COMPRESSION_ASTC);
+                features.insert(wgt::Features::TEXTURE_COMPRESSION_ASTC_SLICED_3D);
                 features.insert(wgt::Features::TEXTURE_COMPRESSION_ASTC_HDR);
             }
         } else {
             features.set(
                 wgt::Features::TEXTURE_COMPRESSION_ASTC,
                 extensions.contains("GL_KHR_texture_compression_astc_ldr"),
             );
+            features.set(
```

Comment:
According to https://github.com/gpuweb/gpuweb/pull/5169, ASTC Sliced 3D feature requires ASTC feature. This code implies that `TEXTURE_COMPRESSION_ASTC_SLICED_3D` while `TEXTURE_COMPRESSION_ASTC`  is not.

See Dawn: https://source.chromium.org/chromium/chromium/src/+/main:third_party/dawn/src/dawn/native/opengl/PhysicalDeviceGL.cpp;l=239;drc=d0afe551ddb31bbab692d18ef43af52329977fbe

Reply:
@beaufortfrancois good catch! I made an update to ensure it checks for both extensions. Does it look better now? PLMK if any further adjustment would improve the code. Thank you!

Reply:
LGTM